example/Procfile

```diff
@@ -1,2 +1,3 @@
 echo: socat TCP4-LISTEN:2000,fork EXEC:cat
 tick: ./bin/tick
+spawn: npx sleep infinity
```

Comment:
we have a helper program called `hang` now that does exactly this & saves us an external dependency on npx

Reply:
Ah the point of npx is to fork and create a new child process. I originally had `bash -c "sleep infinity"` but for some reason that didn't work. Can look into this more if you think it's important.